modules/programs/neovim.nix

```diff
@@ -54,7 +54,10 @@ let
     // optionalAttrs (cfg.plugins != [] ) {
       packages.home-manager.start = map (x: x.plugin or x) cfg.plugins;
     };
-
+  extraMakeWrapperArgs =
+    if builtins.hasAttr "extraPackages" cfg
```

Comment:
you could use `lib.optionalString cf.gextraPackages != []` instead

Reply:
Resolved,should I make any other changes?